ext/standard/math.c

```diff
@@ -898,6 +904,15 @@ PHPAPI int _php_math_basetozval(zval *arg, int base, zval *ret)
 		}
 	}
 
+	if (invalid_chars > 0) {
```

Comment:
You'll probably need some extra handling to make sure that `0xDEADBEEF`, `0b10101001` and `0o640` do not generate a warning.